x/stake/keeper/sdk_types.go

```diff
@@ -57,6 +59,15 @@ func (k Keeper) Validator(ctx sdk.Context, address sdk.AccAddress) sdk.Validator
 	return val
 }
 
+// get the sdk.validator for a particular pubkey
+func (k Keeper) ValidatorByPubKey(ctx sdk.Context, pubkey crypto.PubKey) sdk.Validator {
+	val, found := k.GetValidatorByPubKey(ctx, pubkey)
```

Comment:
Why not use `GetValidatorByPubKey`? Is it to just avoid using a boolean?

Reply:
Not sure I understand the question - `ValidatorByPubKey` is in the `sdk.ValidatorSet` interface, which is used by the slashing module.

Reply:
Apologies, went through the code and realized the interface was updated.